logic/src/commonTest/kotlin/com/wire/kalium/logic/data/sync/SlowSyncRepositoryTest.kt

```diff
@@ -43,7 +42,7 @@ class SlowSyncRepositoryTest {
     }
 
     // TODO: Re-enable once we can update Turbine to 0.11.0+ (Requires Kotlin 1.6.21+)
-    @Ignore
+//     @Ignore
```

Comment:
we can remove the Ignore 🚀

Reply:
Oooh yeah. I was commenting out just to test it and forgot to remove 😄